src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -14,6 +14,8 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using Microsoft.EntityFrameworkCore.Design.Internal;
```

Comment:
@divega How do you feel about this calling internal API? The scenario is being able to load a model in-process at design time. When we create a model visualizer, we'll either need to do the exact same thing, or create a serialization format for models.